test/parallel/test-sync-fileread.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 require('../common');
 const assert = require('assert');
-const fs = require('fs');
 const fixtures = require('../common/fixtures');
 
-assert.strictEqual(fs.readFileSync(fixtures.path('x.txt')).toString(), 'xyz\n');
+assert.strictEqual(fixtures.readSync('x.txt').toString(), 'xyz\n');
```

Comment:
Nit: we can use `fixtures.readSync('x.txt', 'utf8')` instead of calling `toString()`.

Reply:
Oh, very cool. I will make that change as soon as this plane lands.